app/imports/api/actionItems/server/publications.js

```diff
@@ -5,21 +5,36 @@ import { isAdmin } from '/imports/api/users';
 import { Sprints } from '/imports/api/sprints';
 import { ActionItems } from './../ActionItems.js';
 
-Meteor.publish('actionItems', function publishActionItems(sprintId) {
-    check(sprintId, String);
+
+Meteor.publish('actionItems', function publishActionItems(sprintOrProjectId, type = 'sprintId') {
+    check(sprintOrProjectId, String);
+    check(type, String);
 
     const userId = this.userId;
     const isCurrentUserAdmin = isAdmin(userId);
 
```

Comment:
A little bit more DRY:
`const query = type === sprintId ? { sprintId } : { projectId };`
and you can also get in this way `projectId` required to check permissions.